src/AutoMerge/util.jl

```diff
@@ -184,7 +191,7 @@ function comment_text_fail(
         _please_read_these_documents,
         "The following guidelines were not met:\n\n",
         reasons_formatted,
-        _comment_disclaimer(),
+        _comment_disclaimer(; point_to_slack),
```

Comment:
We need to keep Julia 1.3 support.

```suggestion
        _comment_disclaimer(; point_to_slack=point_to_slack),
```

Reply:
Done, let's see if that runs through…